scripts/upgrade

```diff
@@ -18,13 +18,10 @@ app=$YNH_APP_INSTANCE_NAME
 
 domain=$(ynh_app_setting_get --app=$app --key=domain)
 path_url=$(ynh_app_setting_get --app=$app --key=path)
-admin=$(ynh_app_setting_get --app=$app --key=adminusername)
-is_public=$(ynh_app_setting_get --app=$app --key=is_public)
+admin=$(ynh_app_setting_get --app=$app --key=admin)
```

Comment:
new key `admin` should be managed during upgrade from previous versions and key `adminusername` should be removed if you choose to switch from `adminusername` to `admin`

Reply:
I reverted this to as it was.